numbat/modules/core/lists.nbt

```diff
@@ -114,10 +114,39 @@ fn sort_by_key<A, D: Dim>(key: Fn[(A) -> D], xs: List<A>) -> List<A> =
                   sort_by_key(key, drop(floor(len(xs) / 2), xs)),
                   key)
 
-@description("Sort a list of quantities")
+@description("Sort a list of quantities in ascending order")
 @example("sort([3, 2, 7, 8, -4, 0, -5])")
 fn sort<D: Dim>(xs: List<D>) -> List<D> = sort_by_key(id, xs)
 
+fn _negate<D: Dim>(x: D) = -x
+
+@description("Sort a list of quantities in descending order")
+@example("sort_descending([3, 2, 7, 8, -4, 0, -5])")
+fn sort_descending<D: Dim>(xs: List<D>) -> List<D> = sort_by_key(_negate, xs)
```

Comment:
Hm, I'm not sure I like that we add a new function for this here. Once we have closures (and lambdas), this can be easily done with `sort_by_key`. Feel free to move this to the place where you need it, though. Maybe as `_sort_descending` for now.

Reply:
I was debating whether to actually add it here or not. I'll remove it from this PR and add it as you suggested to the relevant PR.